src/load-src.js

```diff
@@ -1,5 +1,49 @@
+var remote = require('remote');
+var app = remote.require('app');
+
 document.addEventListener('DOMContentLoaded', function() {
-    var qs = window.location.search
+    var qs = window.location.search;
     var src = decodeURIComponent(qs.replace('?', '').split('&')[0].split('=')[1]);
-    document.querySelector('#mattermost-remote').setAttribute('src', src);
+    var webview = document.querySelector('#mattermost-remote');
+
+    webview.setAttribute('src', src);
+
+    var badgeUpdateTimer = setInterval(function() {
+        webview.send('unread-count');
+        webview.send('mention-count');
+    }, 1000);
```

Comment:
Polling every second will drive watts up on laptops. We'll have to change this and reverse the modus operandi by pushing from the webview (later, as for now this will do)